server/container_create_linux.go

```diff
@@ -938,6 +944,25 @@ func (s *Server) createSandboxContainer(ctx context.Context, ctr ctrfactory.Cont
 	return ociContainer, nil
 }
 
+func disableFipsForContainer(ctr ctrfactory.Container, containerDir string) error {
+	// Create a unique filename for the FIPS setting file
+	fileName := filepath.Join(containerDir, "sysctl-fips")
+	content := []byte("0\n")
+
+	// Write the value '0' to disable FIPS directly to the file
+	if err := os.WriteFile(fileName, content, 0o644); err != nil {
+		return fmt.Errorf("failed to write to file: %w", err)
+	}
+	ctr.SpecAddMount(rspec.Mount{
+		Destination: "/proc/sys/crypto/fips_enabled",
+		Source:      fileName,
+		Type:        "bind",
+		Options:     []string{"noexec", "nosuid", "nodev", "ro", "bind"},
+	})
```

Comment:
![random](https://github.com/cri-o/cri-o/assets/250723/691bc72a-a076-49f3-96b4-a48909fe7efb)

Reply:
Not sure if I understood this. Do we need to include/exclude anything from `options`?